chronograf/1.10/alpine/Dockerfile

```diff
@@ -25,7 +25,11 @@ RUN set -ex && \
     cp -a /usr/src/chronograf-*/* /usr/bin/ && \
     gpgconf --kill all && \
     rm -rf *.tar.gz* /usr/src /root/.gnupg && \
-    apk del .build-deps
+    apk del .build-deps && \
+    addgroup -S chronograf && \
+    adduser -S chronograf -G chronograf && \
+    mkdir -p /var/lib/chronograf && \
```

Comment:
I suggest `mkdir -m 0750 -p /var/lib/chronograf` so it has the expected permissions. That said, I'm not sure why this directory is being created in the first place since this directory is presumably read-only in the docker environment (and it wasn't used before)?

Reply:
The directory is created beforehand because Chronograf expects this directory to exist. If Chronograf is running without root permissions, it cannot create the directory itself.

The `0750` permission is fixed.